app/Http/Controllers/OrderHistoryController.php

```
@@ -133,7 +163,7 @@ public function orderAttachments(OrderHistory $orderHistory, Order $order): Json
         $attachments = $orderHistory->order->attachments;
 
         return response()->json([
-            'attachments' => AttachmentResource::collection($attachments)
+            'attachments' => AttachmentResource::collection($attachments),
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Http\Resources\AttachmentResource' in method 'orderAttachments'. (undefined)

(StaticAccess)